centos/7/Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM centos:7
 MAINTAINER Roman Tsisyk <roman@tarantool.org>
 
+ARG DEVTOOLSET=devtoolset-8
```

Comment:
I would add an ability to leave enviroment free from devtoolset toolchain. Anyway, it can be done later.